src/CHANGELOG.tsx

```diff
@@ -10,6 +10,7 @@ import Contributor from 'interface/ContributorButton';
 
 // prettier-ignore
 export default [
+  change(date(2020, 10, 29), 'Portuguese localization messages up to line 175', mariogusman),
```

Comment:
You need to import `mariogusman` from `CONTRIBUTORS` before you can use it in that file, thats why the CI is failing.
`CONTRIBUTORS.ts` is exproting a variable named `mariogusman` with all your infos, but the changelog file needs to know where to find this `mariogusman` variable.